models/Asset.php

```diff
@@ -548,6 +548,8 @@ public function save(array $parameters = []): static
 
                     break; // transaction was successfully completed, so we cancel the loop here -> no restart required
                 } catch (Exception $e) {
+                    Logger::error('Unable to save Asset: ' . (string) $e);
+
```

Comment:
would it make sense to move that logging down to line 569? 

because now in case of deadlock exception, we would generate a error and a warn log message, whereas the error log message might be misleading in that case. WDYT?